person.rb

```diff
@@ -13,9 +18,20 @@ def can_use_services?
     of_age? || @parent_permission
   end
 
+  def correct_name
+    @name
+  end
+
   private
 
   def of_age?
     @age >= 18
   end
 end
+
```

Comment:
**`README.md`** File:

- [ **OPTIONAL** ] **IMPORTANT:** What do you think if you improve your `Getting Started` section of your README with other information needed by users in order to use your project locally(such as how to `access` the cloned dir, `use or run` the app, run tests if exixst,...) To help you, I recommend you use this **[Gist_file](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)** as template.